app/views/components/header/_header.html.erb

```diff
@@ -0,0 +1,41 @@
+<header role="banner" id="global-header" class="<%= yield(:header_class) %>">
+  <div class="header-wrapper">
+    <div class="header-global">
+      <div class="header-logo">
+        <a href="<%= content_for?(:homepage_url) ? yield(:homepage_url) : "https://www.gov.uk/" %>"
```

Comment:
In this context, is the `:homepage_url` and `https://www.gov.uk/` the same thing?

Reply:
Hmm, seems to be used in the previous header - so maybe we should leave this be.